<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPasswordDialog</class>
 <widget class="QDialog" name="ForgotPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_salary">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newPassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_resetPassword">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset pass</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>salaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nv mdp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
